drain.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Ansible playbook to drain Slurm compute nodes. Waits for compute nodes to be
+# drained for up to a day.
+#
+# Variables:
+# - nodes_to_drain: list of compute nodes to drain
+
+- name: Drain compute nodes
+  become: yes
+  shell: "scontrol update nodename={{ item }} state=DRAIN reason='maintenance'"
```

Comment:
Consider using `command` in place of `shell` if only because the process group is slightly smaller.

Reply:
Thanks for the advice.